test/e2e/e2e_test.go

```diff
@@ -249,6 +248,44 @@ var _ = Describe("e2e", func() {
 			Expect(err).To(BeNil())
 		})
 	})
+
+	Context("Network issues", func() {
+		conformanceTest := func() {
+			// when
+			_, err = deployment.Create(edgedeployemntDeviceId("nginx", device.GetId(), hostPort, nginxPort))
+			Expect(err).To(BeNil())
+
+			// then
+			// Check the edgedevice report proper state of workload:
+			err = device.WaitForDeploymentState("nginx", "Running")
+			Expect(err).To(BeNil())
+
+			// Check the nginx is serving content:
+			stdout, err := device.Exec(fmt.Sprintf("curl http://localhost:%d", hostPort))
+			Expect(err).To(BeNil())
+			Expect(stdout).To(ContainSubstring("Welcome to nginx!"))
+		}
+
+		It("Packets delayed 200ms", func() {
+			// given
+			err := device.Register(
+				"dnf install iproute -y",
```

Comment:
Do we want to add this to base test image?

Reply:
should we add that for two test cases? I'm not sure about :sweat_smile:

Reply:
Well, not really, but one day when we would wanna improve performance I think such things we will move ;)

Reply:
When we'll start working on performance of the device-worker we might need additional packages.
In that case, it would be indeed faster to install all tests prerequisites during image creation time vs at test time.
Also, it from some reason, we'd like to perform the tests against ostree-based image, then this will fail.
But we can delay this to a future date when it will be needed.